architecture/index.adoc

```diff
@@ -0,0 +1,71 @@
+
+toc::[]
+
+{product-title} is a cloud-based Kubernetes container platform.
+The foundation of {product-title} is based on Kubernetes and therefore shares the same technology.
+To learn more about {product-title} and Kubernetes, refer to xref:../architecture/architecture.adoc#architecture[product architecture].
+
+[id="about-installation-and-updates"]
+== About installation and updates
+
+As a cluster administrator, you can use the {product-title} xref:../architecture/architecture-installation.adoc#architecture-installation[installation program] to install to deploy a cluster on:
+
+** Installer-provisioned infrastructure clusters (IPI)
+** User-provisioned infrastructure clusters (UPI)
+
+[id="About-control-planes"]
+== About control planes
+
+The xref:../architecture/control-plane.adoc#control-plane[control plane] manages the worker nodes and the pods in your cluster. You can configure nodes with the use of machine config pools (MCP).
+Machine config pools are groups of machines such as, control plane components or user workloads. These groups are based on the resources handled.
+{product-title} assigns hosts different roles. These roles define the function of a machine in a cluster.
+The cluster contains definitions for the standard master and worker role types.
+
+You can use operators to package, deploy, and manage services on the control plane. With operators, you can use `kubectl` and `oc` commands.
+Operators are important components in {product-title} because they:
+
+* Perform health checks.
+* Enable the way to watch an application.
+* Manage over-the-air updates.
+* Ensure applications stay in the specified state.
+
+[id="how-to-develop-your-containerized-application"]
+== How to develop your containerized application
+
+As a developer, you can xref:../architecture/understanding-development.adoc#understanding-development[develop your containerized application] in many ways and there are different approaches based on requirements.
+You can use different tools, methods, and formats to develop your containerized application. You can build a simple container application with the various container build tool, base image, and registry options.
+You can use supporting components such as OperatorHub and templates to develop your application.
+If you want to make your application, you can package and deploy your application as an Operator.
+
+You can also create a Kubernetes manifest and store it in a Git repository.
+Kubernetes works on basic units called `pods`. Pods are the next step in application development.
+A service is a grouping of a logical set of pods and access policies. Services provide permanent internal IP addresses and host names for other applications to use as pods are created and destroyed.
+Based on the type of your application, Kubernetes defines a specific type of workload.
+
+[id="coreOS"]
+== All you need to know about Red Hat Enterprise Linux CoreOS (RHCOS)
+
+As a cluster administrator, you can
```

Comment:
This part in general needs some rework. The lead-in sentence shouldn't be fragmented, so something like, "As a cluster administrator, you can perform the following tasks" would be better. But in the list itself, the tense doesn't match. And also, it's a lot of bullets for basically one statement, which is that "cluster administrators can configure and deploy RHCOS on either installer-provisioned infrastructure or user-provisioned infrastructure", right?

And since RHCOS provides Ignition files, I'd consider combining those two sections into one. Something simple for a header would probably suffice, such as, "RHCOS and Ignition"